### DISCUSSION TITLE
Delete Unused username

### DISCUSSION THREAD

**Discussion by bolez** [community/community#135952] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it possible to delete all unused usernames? We’ve noticed that many accounts were created without any activity. If you could delete these inactive accounts or free up those usernames, it would be very helpful. This would allow new users to choose shorter and more desirable usernames, as many of the preferred ones are currently taken, forcing new users to settle for longer usernames.

**Comment by ebndev** [comment]
Hi @harnya,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
